llvm/lib/Target/AIE/AIEPostPipeliner.cpp

```diff
@@ -270,8 +300,15 @@ bool PostPipeliner::scheduleFirstIteration() {
     LLVM_DEBUG(dbgs() << "  Emit in " << -Depth + LocalCycle << "\n");
     int Cycle = -Depth + LocalCycle;
     LLVM_DEBUG(dbgs() << "  Emit in " << Cycle << "\n");
-    HR.emitInScoreboard(Scoreboard, MI->getDesc(), MemoryBanks, MI->operands(),
-                        MI->getMF()->getRegInfo(), Cycle);
+    for (int N = 0; N < NCopies; N++) {
```

Comment:
nit: I would use unsigned here ;-).

Reply:
I'm sorry, I solemnly pledged to only use unsigned for bitmasks (and perhaps very small types like char for lookup tables)